com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -81,6 +81,13 @@ private static string GetSceneEventTypeName(uint typeCode)
         };
 #endif
 
+#if MULTIPLAYER_TOOLS_1_0_0_PRE_7
+        private readonly Gauge m_NetworkObjectsGauge = new Gauge(NetworkMetricTypes.NetworkObjects.Id)
+        {
+            ShouldResetOnDispatch = true,
+        };
+#endif
+
```

Comment:
Maybe put this under the same #if as above, seeing as the condition is the same

Reply:
Oh right, it makes sense now. It didn't before.